src/components/layout/layout.js

```diff
@@ -206,8 +207,8 @@ const Layout = ({
             FooterNav = <Footer />
             break
         case 'landing-page':
-            Navigation = <Nav hide_signup_login={true} no_language={true} />
-            FooterNav = <Footer />
+            Navigation = <NavInterim no_button={true} no_languageSwitcher={no_languageSwitcher} />
```

Comment:
Instead of doing this, can we instead pass something like this

```jsx
<NavInterim landing_type />
```
and inside `nav.js` for NavInterim, you can consume it like this

```jsx
  NavInterim = ({ interim_type, landing_type }) => ( .... )
```
This will avoid having you to add the no_language_switcher prop for all landing-page layouts.

Reply:
thanks for the suggestion, done

Reply:
thanks, done